examples/01_plotting/plot_demo_plotting.py

```diff
@@ -20,10 +21,12 @@
 haxby_dataset = datasets.fetch_haxby()
 
 # print basic information on the dataset
-print('First subject anatomical nifti image (3D) is at: %s' %
-      haxby_dataset.anat[0])
-print('First subject functional nifti image (4D) is at: %s' %
-      haxby_dataset.func[0])  # 4D data
+print("First subject anatomical nifti image (3D) is "
+      f"at: {haxby_dataset.anat[0]}")
+print(
+    "First subject functional nifti image (4D) is "
+    f"at: {haxby_dataset.func[0]}"
+)
```

Comment:
These print statements can be formatted the same, no?

Reply:
actually this is one of the file that is also in another PR: https://github.com/nilearn/nilearn/issues/2528#issuecomment-1414489207

so I did not run black on it to avoid increasing the number of merge conflicts

hence why the formatting is different

As the PR with those files get merged, I am planning to slowly re-include into the list of files pre-commit can act on and then reformat them